qualitycontrol.sh

```diff
@@ -20,19 +21,23 @@ incremental_recipe: |
 #!/bin/bash -ex
 
 case $ARCHITECTURE in
-    osx*) [[ ! $BOOST_ROOT ]] && BOOST_ROOT=$(brew --prefix boost);;
+  osx*)
+    [[ ! $BOOST_ROOT ]] && BOOST_ROOT=$(brew --prefix boost)
+    [[ ! $GSL_ROOT ]] && GSL_ROOT=`brew --prefix gsl` ;;
```

Comment:
`GSL_ROOT` is not `MS_GSL_ROOT`. They are two different libraries (blame Microsoft for the aliasing...).. Is this what you really want????

Reply:
Sorry, I did not know that, I copy-pasted it without enough understanding. Fixed.